deployments/deployments.goerli.json

```diff
@@ -0,0 +1,8 @@
+{
+ "FuelSidechainConsensus": "0xF712e555ce59858f680890DA7dc2B51eD048580d",
+ "FuelMessagePortal": "0x4c7181fff2053D232Fc74Ff165b83FE8Dcb65910",
+ "L1ERC20Gateway": "0x96c53cd98B7297564716a8f2E1de2C83928Af2fe",
+ "FuelSidechainConsensus_impl": "0xF712e555ce59858f680890DA7dc2B51eD048580d",
+ "FuelMessagePortal_impl": "0x4c7181fff2053D232Fc74Ff165b83FE8Dcb65910",
+ "L1ERC20Gateway_impl": "0x96c53cd98B7297564716a8f2E1de2C83928Af2fe"
+}
```

Comment:
Should these continue to use duplicate addresses or are these meant to be updated prior to merging?

Reply:
This file check-in is meant to be a reference to currently deployed contracts. Eventually there will be a mainnet one too. There are duplicates for now, but only because the current contracts on Goerli are not upgradeable.